### PR TITLE
fix: improved the ui of schedule fragment

### LINKED ISSUE #1738: Improvement in the UI of Schedule Fragment
**Actual Behaviour**

There are certain points on improvijg the UI of schedule fragment.

1. removing the session description. that is only showing the title along with time and location.

2. show length ( end time - start time ) instead of the start time and end time.

3. show the time 24 hour based and remove the AM and PM.

4. add a video button along with the track imageview.

5. remove sorting option and replace it with the filter used as a floating action button as of now.

**Expected Behaviour**

Making UI changes to the fragment.

**Steps to reproduce it**

-

**LogCat for the issue**

-

**Screenshots of the issue**

-

**Would you like to work on the issue?**

-

**Target Milestone to solve the issue**

https://github.com/fossasia/open-event-android/milestone/31 

@iamareebjamal @Shailesh351 @shreyasnbhat @rohanagarwal94 
please check which points seem apt to you.

### PR DESCRIPTION
**Pull request by arp95**
Fixes #1738 
Changes: 

improved the ui of schdule fragment and also made location name bold in location fragment.

Screenshots for the change: 

screenshot_20170630-202201

@iamareebjamal @Shailesh351 @shreyasnbhat @rohanagarwal94 
please review.

### CODE REVIEW

**Comment by Shailesh351** [comment]
IMO bold text don't look good. You can increase text size little bit.

File: android/app/src/main/res/layout/item_schedule.xml [hunk 1]
```diff
@@ -29,19 +29,9 @@
             android:paddingTop="@dimen/padding_small"
             android:textColor="@color/black"
             android:textSize="@dimen/text_size_large"
+            android:textStyle="bold"
             tools:text="Slot Title" />
 
-        <TextView
```
Comment (Shailesh351) [inline android/app/src/main/res/layout/item_schedule.xml]: also remove session description field from java file

File: android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java [hunk 2]
```diff
@@ -71,17 +67,14 @@ public DayScheduleViewHolder(View itemView,Context context) {
...
+        String endTimeText = DateUtils.formatDateWithDefault(DateUtils.FORMAT_24H, session.getEndTime());
         String title = Utils.checkStringEmpty(session.getTitle());
-        String shortAbstract = Utils.checkStringEmpty(session.getShortAbstract());
 
         startTime.setText(startTimeText);
         endTime.setText(endTimeText);
         slotTitle.setText(title);
 
-        Views.setHtml(slotDescription, shortAbstract, true);
-
```
Comment (arp95) [inline android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java:84]: @Shailesh351 i removed over here. fine right?

Comment (Shailesh351) [reply android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java:84]: Ohh sorry my bad.

**Comment by shreyasnbhat** [comment]
@arp95 I feel that the bold looks good as it highlights the event name.
Lets wait for others.

**Comment by arp95** [comment]
@shreyasnbhat 
sure. i have changed the bold so if anyone else prefers it do tell.

**Comment by arp95** [comment]
@iamareebjamal @rohanagarwal94 
your thoughts?

**Comment by arp95** [comment]
Guys please review.

**Comment by arp95** [comment]
@iamareebjamal @Shailesh351 @rohanagarwal94
please review.

**Comment by rohanagarwal94** [comment]
@arp95 I agree with @Shailesh351

**Comment by arp95** [comment]
so i have increased the text size and removed bold text.
please approve now. @Shailesh351 @rohanagarwal94 @shreyasnbhat

**Review by the-dagger — APPROVED** [review]

**Comment by arp95** [comment]
@the-dagger 
please merge.

**Pull request merged**